src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -298,17 +298,42 @@ public void Initialize()
             }
         }
 
-        public async Task PutEditVersioningCommandAsync(JsonOperationContext context, string databaseName, BlittableJsonReaderObject val)
+        public async Task<long> ModifyDatabaseExpirationBundle(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson)
```

Comment:
Probably can create a base class /interface and then just have a SendCommand method